sample_logs/graphdb/create_graph.cypher

```diff
@@ -0,0 +1,98 @@
+CREATE (p1:Person {name: 'John Doe', birth_date: '1980-01-01'})
```

Comment:
Do we need to document how somebody would run this script to populate their neo4j database with this data? Maybe in the `data` README.

Reply:
we could even auto-import the script when the docker container starts-up?

https://stackoverflow.com/questions/48357238/how-can-i-run-cypher-scripts-on-startup-of-a-neo4j-docker-container

Reply:
Sounds good, I'm adding some instructions in the `data` README